Automattic/wp-calypso
# Remove check-npm-client

### Background

When we migrated to `yarn` a few months ago, we introduced `check-npm-client` (in #40893) to help developer with the transition and make sure nobody run `npm` accidentally.

We have been with `yarn` for months and I don't think someone running `npm` and go unnoticed is a likely scenario. I think we can get rid of `check-npm-client` to reduce the noise and avoid some overhead of checking for `yarn` over and over again (specially when running chained npm scripts).

### Changes

* Remove `check-npm-client`.

### Testing 

* Checkout the branch and ensure your usual devloop (eg: `yarn && yarn start`) is still working
* Ensure tests are passing
